tools/spinel-cli/SNIFFER.md

```diff
@@ -94,3 +94,15 @@ This will connect to stock openthread ncp firmware over the given UART,
 make the node into a promiscuous mode sniffer on the given channel, 
 open up wireshark, and start streaming packets into wireshark. 
 
+## Troubleshooting
+Q: sniffer.py throws ```ImportError: No module named dnet``` on OSX
+A: install the libdnet package for OSX -
```

Comment:
nit: the question and answer appear on the same line in the actual render.  Would be more readable if they were on separate lines.